tests/photoCarousel.test.js

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable no-undef */
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import {
+  LeftArrowContainer, RightArrowContainer, PhotosSection,
+} from '../client/styledComponents/styledPhotoCarousel';
+
+import PhotoCarousel from '../client/components/photoCarousel';
+import Photo from '../client/components/photo';
+import { PhotoContainer } from '../client/styledComponents/styledPhoto';
+
+const photo1 = {
+  photo_id: 1,
+  user_id: 11,
+  medium: 'https://kittenoverload.com/images/a907e923c23ff653673188e11c2f87c2/image.jpg',
+  posted: '1982-07-13',
+  caption: 'This is a photo cat-tion',
+  label: 'Label',
+  first_name: 'Julie',
+  last_name: 'Hambersam',
+  helpful: 1982,
+  not_helpful: 713,
+};
+
+const photo2 = {
+  photo_id: 2,
+  user_id: 11,
+  medium: 'https://kittenoverload.com/images/a907e923c23ff653673188e11c2f87c2/image.jpg',
+  posted: '1982-07-13',
+  caption: 'This is a photo cat-tion',
+  label: 'Label',
+  first_name: 'Julie',
+  last_name: 'Hambersam',
+  helpful: 1982,
+  not_helpful: 713,
+};
+
+const photo3 = {
+  photo_id: 3,
+  user_id: 11,
+  medium: 'https://kittenoverload.com/images/a907e923c23ff653673188e11c2f87c2/image.jpg',
+  posted: '1982-07-13',
+  caption: 'This is a photo cat-tion',
+  label: 'Label',
+  first_name: 'Julie',
+  last_name: 'Hambersam',
+  helpful: 1982,
+  not_helpful: 713,
+};
+
+const photo4 = {
+  photo_id: 4,
+  user_id: 11,
+  medium: 'https://kittenoverload.com/images/a907e923c23ff653673188e11c2f87c2/image.jpg',
+  posted: '1982-07-13',
+  caption: 'This is a photo cat-tion',
+  label: 'Label',
+  first_name: 'Julie',
+  last_name: 'Hambersam',
+  helpful: 1982,
+  not_helpful: 713,
+};
+
+const photos = [photo1, photo2, photo3, photo4];
+
+describe('Photo Carousel Component', () => {
+  describe('Appearance', () => {
+    const wrapper = shallow(<PhotoCarousel photos={photos} />);
+    test('it should display 3 photos', () => {
+      const photoComponent = wrapper.find('Photo');
+      expect(photoComponent).toHaveLength(3);
+    });
+
+    test('it should have a left navigation arrow', () => {
+      const full = wrapper.find(PhotosSection);
+      expect(full).toHaveLength(1);
+    });
+
+    test('it should have a right navigation arrow', () => {
+      const rightArrow = wrapper.find(RightArrowContainer);
+      expect(rightArrow).toHaveLength(1);
+    });
+  });
+
+  describe('Functionality', () => {
```

Comment:
Spacing looks like it may be a bit off here for this describe block

Reply:
It's nested inside another describe block, so it should be fine.